src/main/java/org/broadinstitute/hellbender/tools/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -48,6 +49,10 @@
     
     private static final Logger log = LogManager.getLogger();
 
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME, doc = "File to write the output to.")
+    public File OUTPUT;
```

Comment:
Agreed, but the only change here was to push the definition down from SinglePassSam into the individual classes as an interim step until these classes are all ported to the new framework. If we change it to be lower case, all of the references here and in all of the other classes it got pushed down to would also have to be changed. And this class also has 5 other fields all in upper case, and doesn't conform in other ways. I'd propose we wait and make those changes in the PRs where these tools and classes are made GATK compliant, rather than adding all of those changes to this PR.